modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -180,10 +178,10 @@ RankingPortletDisplayContext rankingPortletDisplayContext = (RankingPortletDispl
 					<c:when test="<%= Validator.isNotNull(rankingEntryDisplayContext.getSXPBlueprintExternalReferenceCode()) %>">
 
 						<%
-						SXPBlueprint sxpBlueprint = SXPBlueprintLocalServiceUtil.fetchSXPBlueprintByExternalReferenceCode(rankingEntryDisplayContext.getSXPBlueprintExternalReferenceCode(), themeDisplay.getCompanyId());
+						String sxpBlueprintTitle = rankingEntryDisplayContext.getSXPBlueprintTitle();
 						%>
 
-						<span class="lfr-portal-tooltip" data-title='<%= Validator.isNotNull(sxpBlueprint) ? HtmlUtil.escape(sxpBlueprint.getTitle(locale)) : LanguageUtil.get(request, "the-blueprint-associated-with-this-ranking-was-deleted") %>'>
+						<span class="lfr-portal-tooltip" data-title='<%= Validator.isNotNull(sxpBlueprintTitle) ? HtmlUtil.escape(sxpBlueprintTitle) : LanguageUtil.get(request, "the-blueprint-associated-with-this-ranking-was-deleted") %>'>
```

Comment:
if LES is only deactivated then "the-blueprint-associated-with-this-ranking-was-deleted" might not be completely accurate any more... maybe the display context will need a method for checking if LES is enabled/disabled... but this could probably be handled later if we do the additional "edge case" ticket we were talking about